package.json

```diff
@@ -3,7 +3,7 @@
     "version": "2022.1.9-0",
     "description": "Materials Designer",
     "scripts": {
-        "start": "cross-env PORT=3001 SKIP_PREFLIGHT_CHECK=true react-scripts start",
+        "start": "cross-env PORT=3001 SKIP_PREFLIGHT_CHECK=true react-scripts start --experimental-wasm-bigint",
```

Comment:
What is this change for?

Reply:
As explained in a call -- it's necessary for wasm in pyodide to work with Node 14

Reply:
You're right. Removed right now -- no effect.